src/commands/synthetics/README.md

```diff
@@ -232,6 +232,12 @@ The `run-tests` sub-command accepts the `--public-id` (or shorthand `-p`) argume
 yarn datadog-ci synthetics run-tests --public-id pub-lic-id1 --public-id pub-lic-id2
 ```
 
+You can also specify a specific version of a test using the format `<public_id>@<version>`:
+
+```bash
+yarn datadog-ci synthetics run-tests --public-id pub-lic-id1@2 --public-id pub-lic-id1@4
```

Comment:
```suggestion
yarn datadog-ci synthetics run-tests --public-id aaa-aaa-aaa@2 --public-id bbb-bbb-bbb@4
```

Reply:
I didn't use the `abc-def-ghi` and `123-456-789` examples like [in this section](https://github.com/DataDog/datadog-ci/blob/master/src/commands/synthetics/README.md#publicids) because `123-456-789@4` looks a bit weird 😅